app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -332,6 +332,12 @@ private void notifyAndMigrateDataSetObservers() {
         for (DataSetObserver observer : observersWaitingForLoad) {
             observer.onChanged();
         }
+
+        if (!ConfigUtils.isBetaFlavour()) {
```

Comment:
I would suggest removing the `!` and inverting the two sections, for increased readability.

Reply:
Done